server_admin/topics/clients/protocol-mappers.adoc

```diff
@@ -15,7 +15,9 @@ an OIDC based client.
 .Mappers Tab
 image:{project_images}/mappers-oidc.png[]
 
-Each client has several built-in mappers that are created for it by default.  They map things like, for example, email address to
+The new client doesn't have any built-in mappers, however it usually inherits some mappers from the client scopes as
+described in <<_client_scopes, client scopes section>>.
+Protocol mappers map things like, for example, email address to
 a specific claim in the identity and access token.  Their function should each be self explanatory from their name.  There
```

Comment:
The new client does not have any built-in mappers, however it usually inherits some mappers from the client scopes as described in the <<_client_scopes, client scopes section>>. Protocol mappers map things like, for example, email address to